archey/entries/model.py

```diff
@@ -23,7 +23,8 @@ def __init__(self, *args, **kwargs):
             or self._fetch_dmi_info() \
             or self._fetch_sysctl_hw() \
             or self._fetch_raspberry_pi_revision() \
-            or self._fetch_android_device_model()
+            or self._fetch_android_device_model() \
```

Comment:
Maybe we could run your new specific method "sooner", like just after (or even before) `_fetch_sysctl_hw`, WDYT ?

Reply:
Yes, I think it's right, let's avoid to run "useless" code.  Before `_fetch_sysctl_hw` sounds good.

Alternatively instead of relying on the `lazy` or conditions, we could test using ```platform.system()``` or it could be kept for you as a refactoring exercise :)

Reply:
> Alternatively instead of relying on the `lazy` evaluations, we could test using `platform.system()` or it could be left for you as a refactoring exercise slightly_smiling_face

You're right, something like this could be achieved in this entry too ; I'll do it for the next version afterwards :pray:

Reply:
(I've noted the action [here](https://github.com/HorlogeSkynet/archey4/projects/4#card-85429026))